### PR TITLE
feat: UseOpCompleter (related to #6264)

### PR DESCRIPTION
**Pull request by xdesou01**

### CODE REVIEW

**Comment by magnus-madsen** [comment]
Conflicted.

**Comment by xdesou01** [comment]
> Conflicted.

Resolved

File: main/src/ca/uwaterloo/flix/api/lsp/provider/completion/UseCompleter.scala [hunk 1]
```diff
@@ -24,6 +24,6 @@ object UseCompleter extends Completer {
     * Returns a List of Completion for completer.
     */
   override def getCompletions(context: CompletionContext)(implicit flix: Flix, index: Index, root: TypedAst.Root, delta: DeltaContext): Iterable[Completion] = {
-    UseModuleCompleter.getCompletions(context) ++ UseEnumCompleter.getCompletions(context) ++ UseEffCompleter.getCompletions(context) ++ UseDefCompleter.getCompletions(context) ++ UseSignatureCompleter.getCompletions(context)
+    UseModuleCompleter.getCompletions(context) ++ UseEnumCompleter.getCompletions(context) ++ UseEffCompleter.getCompletions(context) ++ UseDefCompleter.getCompletions(context) ++ UseSignatureCompleter.getCompletions(context) ++ UseOpCompleter.getCompletions(context)
```
Comment (magnus-madsen) [inline main/src/ca/uwaterloo/flix/api/lsp/provider/completion/UseCompleter.scala:27]: Maybe its time to break this into multiple lines, with one completer per line.

**Comment by magnus-madsen** [comment]
Do you have a screenshot?

**Comment by xdesou01** [comment]
> Do you have a screenshot?

image
image
image

**Review by magnus-madsen — APPROVED** [review]

**Pull request merged**